src/test/api/users.test.ts

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import fetchApiData from '../../api/users';
+
+jest.mock('axios');
+
+describe('fetchApiData', () => {
+
+    // reset all mocked functions' states after each test
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    const responseData = [{ id: 1, title: 'Test Post' }];
+    const response = { data: responseData };
+
+    it('fetches successfully data from API', async () => {
+        (axios.get as jest.Mock).mockResolvedValue(response);
+
+        const data = await fetchApiData();
+        expect(data).toEqual(responseData);
+    });
+
+    it('handles API error', async () => {
+        const errorMeassage = 'Network Error';
+        (axios.get as jest.Mock).mockRejectedValue(new Error(errorMeassage));
+
+        try {
```

Comment:
instead of adding try catch and if condition inside test , please check any other alternative is available